hledger-lib/Hledger/Data/TransactionModifier.hs

```diff
@@ -47,7 +47,7 @@ import Hledger.Utils.Debug
 -- 0000/01/01
```

Comment:
"Restore ability..." - makes me wonder when they lost that ability. Was it in the previous commit, & so should this be merged with that ? Or was it an older one we already merged ?